src/mjolnir/util.cc

```diff
@@ -277,17 +279,31 @@ bool build_tile_set(const boost::property_tree::ptree& config,
       osm_data.write_to_temp_files(tile_dir);
     }
   }
+  // Construct edges
+  std::map<baldr::GraphId, size_t> tiles;
+  if (start_stage <= BuildStage::kConstructEdges && BuildStage::kConstructEdges <= end_stage) {
+    tiles = GraphBuilder::BuildEdges(config, osm_data, ways_bin, way_nodes_bin, nodes_bin, edges_bin);
+    // Output manifest
+    TileManifest manifest{tiles};
+    manifest.LogToFile(tile_manifest);
+  }
 
   // Build Valhalla routing tiles
   if (start_stage <= BuildStage::kBuild && BuildStage::kBuild <= end_stage) {
     // Read OSMData from files if building tiles is the first stage
     if (start_stage == BuildStage::kBuild) {
+      if (filesystem::exists(tile_manifest)) {
```

Comment:
This here needs to also check that the output from `::BuildEdges` exists.

I don't know if we want to do some other data consistency check here.